src/main/java/mia/Mia.java

```diff
@@ -0,0 +1,66 @@
+package mia;
+
+import general.ui.ChatWindow;
+import general.ui.Span;
+
+/**
+ * An instance of {@code Mia}, a virtual assistant.
+ */
+public class Mia {
+    private static final String logo = "┏━━━━━━━━━━━━━━━━━━━━━━━━━━━┓\n"
+                                     + "┃ You are talking to MIA... ┃\n"
+                                     + "┗━━━━━━━━━━━━━━━━━━━━━━━━━━━┛\n";
+    private final TaskManager tasksManager;
+    private final ChatWindow window = new ChatWindow(50);
+
+    public Mia(String dataPath) {
+        tasksManager = new TaskManager(dataPath);
+    }
+
+    /**
+     * Sends a message to the user.
+     *
+     * @param message the message to be sent
+     */
+    public void respond(String message) {
+        window.printResponse(new Span(message));
+    }
+
+    /**
+     * Gets the {@code TaskManager} task list manager for this {@code Mia} instance.
+     *
+     * @return The {@code TaskManager} associated with this {@code Mia} instance.
+     */
+    public TaskManager getTasks() {
```

Comment:
Why not name it getTaskManager() to better identify return type?

Reply:
Because I felt that the return type should not influence the method name. In my opinion, the method name should return what the object _is_ (e.g. "get my laptop"), in natural language (to be as clear as possible), as opposed to specifying the exact specification (e.g. "get my HP Book Series-12403", as this is provided from the method signature anyway.

What do you think?

_PS: The HP model is fake (I think)_